src/website/shared/channels.py

```diff
@@ -19,6 +19,7 @@ class NixChannelChannel(TriggerChannel):
 @dataclass
 class NixEvaluationChannel(TriggerChannel):
     model = NixEvaluation
+    lock_notifications = True
```

Comment:
What's this?

Reply:
I will add a comment but this enables processing of notifications exactly once except of potentially random workers grab the notification and work through it.